src/layman/error_list.py

```diff
@@ -53,7 +53,7 @@
     50: (500, 'Error when normalizing raster file'),
     51: (500, 'Error when generating thumbnail'),
     52: (400, 'GeoServer HTTP or connection error'),
-    53: (500, 'Error when publishing on GeoServer. It happens for example for raster files with wrong explicit CRS.'),
+    53: (500, 'Wrong bounding box after raster normalization. It happens for example when wrong explicit CRS was set.'),
```

Comment:
I think, that the error used to be more broad. It can now be fired from `refresh_wfs` task, probably with different core reason than wrong bbox.